### PR TITLE
2.33 doc review

### PR DESCRIPTION
**Pull request by camillemonchicourt**
I tried to complete some parts with my experience with the 2.33 version.

- About configuration, I tried to clarify it with a basic one and an advanced one, but not perfect yet
- About ``anonymize_reports`` command, I changed the ``.bin/django`` by ``sudo geotrek`` but it doesn't seem to be right

### CODE REVIEW

**Comment by camillemonchicourt** [comment]
Also, I couldn't find where to set the Emails settings (as explained in actual documentation - https://geotrek.readthedocs.io/en/master/configuration.html#email-settings) that was previously in ``etc/settings.ini`` file, but is not in Basic configuration now, nor in advanced configuration (https://github.com/GeotrekCE/Geotrek-admin/blob/master/geotrek/settings/base.py).

**Comment by camillemonchicourt** [comment]
Should I cut this PR in smaller ones ?
And I can't find where to set the email settings. It is in ``custom.py`` now ?
Thanks.

**Comment by camillemonchicourt** [comment]
Pour la configuration des emails, c'est dans la surcouche de la configuration avancées (custom.py), l'exemple est dans le template : https://github.com/GeotrekCE/Geotrek-admin/blob/master/geotrek/settings/custom.py.dist#L38

**Comment by numahell** [comment]
Hello @camillemonchicourt , can I rebase you branch onto master ? I have other things to add to documentation.

**Comment by camillemonchicourt** [comment]
Yes yes this doc review is a bit old now so I let you see if it you can rebase it. Otherwise I will do a new doc review later.

**Review by numahell — APPROVED** [review]

File: docs/faq.rst [hunk 1]
```diff
@@ -118,10 +118,10 @@ the source code is probably the least common action. For example :
 * Write and talk about *Geotrek*, at conferences, workgroups, forums ;
 
 * Translate the documentation ;
-* Translate the menus, buttons and labels (we use `Transifex <https://www.transifex.com/organization/makina-corpus>`_) ;
+* Translate the menus, buttons and labels (we use `Weblate <https://weblate.makina-corpus.net>`_) ;
```
[the remark this answers is not in the document]
Comment (numahell) [reply docs/faq.rst:121]: Actually it is now official

**Pull request merged**